Source/Classes/SIGPlugin.m

```diff
@@ -571,7 +571,7 @@ - (void)openFileOnGitHub:(id)sender
     {
         // If the repo path does not include a bitbucket server, we assume a github server. Consequently we can
         // support GitHub enterprise instances with arbitrary server names.
-        path = [NSString stringWithFormat:@"/blob/%@/%@#L%ld-%ld",
+        path = [NSString stringWithFormat:@"/blob/%@/%@#L%ld-L%ld",
```

Comment:
I just tested it, you're right! I wonder how that ever worked? Did GitHub change the format? I think so...

Thanks a lot! :+1: